### ISSUE TITLE
Upload gpx in multi-gpx monitor route broken

### ISSUE THREAD

**Issue by vmarc** [vmarc/knooppuntnet#393]
Error parsing json server response when opening the upload page:

```
JSON.parse: unexpected character at line 1 column 1 of the JSON data
```

**Comment by pelderson** [comment]
I don't know if this is the same issue? When trying to upload a segment gpx nothing happens. F12 in Chrome gives:
![image](https://github.com/vmarc/knooppuntnet/assets/38458314/f9b3769c-cdfd-4053-a519-48bb01a3cb01)

**Comment by vmarc** [comment]
Fixed in release v4.3.36.
